modules/apps/commerce/commerce-notification-service/src/main/java/com/liferay/commerce/notification/service/impl/CommerceNotificationTemplateLocalServiceImpl.java

```diff
@@ -135,8 +135,8 @@ public CommerceNotificationTemplate deleteCommerceNotificationTemplate(
 
 		// Commerce notification template account groups rels
 
-		commerceNotificationTemplateCommerceAccountGroupRelLocalService.
-			deleteCNTemplateCommerceAccountGroupRelsByCommerceNotificationTemplateId(
+		commerceNotificationTemplateCommerceAccountGroupRelPersistence.
+			removeByCommerceNotificationTemplateId(
 				commerceNotificationTemplate.
```

Comment:
@JanisZhang, this is not circular reference issue. commerceNotificationTemplateCommerceAccountGroupRelLocalService is not available after **gw buildService**. You may directly add Reference:CommerceNotificationTemplateCommerceAccountGroupRelLocalService to CommerceNotificationTemplateLocalServiceImpl class, instead of using commerceNotificationTemplateCommerceAccountGroupRelPersistence.

cc @dantewang